docs/docusaurus/docs/templates/how_to_template.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: TEMPLATE How to guide {stub}
 ---
-import Prerequisites from '../connecting_to_your_data/components/prerequisites.jsx'
-import NextSteps from '../connecting_to_your_data/components/next_steps.md'
+import Prerequisites from '/docs/guides/connecting_to_your_data/components/prerequisites.jsx'
```

Comment:
```suggestion
import Prerequisites from '/docs/components/_prerequisites.jsx'
```
This is an old link.  We're trying to consolidate "universal" components like this in `/docs/components/` so we can more easily update formatting for them.

The same applies to the "NextSteps" import, except that I don't believe we have an updated version in `/docs/components/` yet.